v4-client-py/v4_client_py/clients/constants.py

```diff
@@ -1,75 +1,63 @@
 from enum import Enum
-from ..chain.aerial.config import NetworkConfig
 
-
-# ------------ API URLs ------------
-INDEXER_API_HOST_MAINNET = None
```

Comment:
it would be great if there is one example for mainnet, because examples work great on testnet but then all kind of DNS issues come up for mainnet